packages/xrpl/src/models/transactions/common.ts

```diff
@@ -9,15 +9,15 @@ import {
   AuthorizeCredential,
   Currency,
   IssuedCurrencyAmount,
+  MPTAmount,
   Memo,
   Signer,
   XChainBridge,
-  MPTAmount,
 } from '../common'
 import { onlyHasFields } from '../utils'
 
 const MEMO_SIZE = 3
-const MAX_CREDENTIALS_LIST_LENGTH = 8
+export const MAX_CREDENTIALS_ARRAY_LENGTH = 8
```

Comment:
```suggestion
export const MAX_AUTHORIZED_CREDENTIALS = 8
```

Reply:
@mvadari  I don't have strong preference about your variable-name-change suggestions, but they are used in multiple places across the codebase. Furthermore, these names are suited to the subjective preferences of the author.

If you could provide me a commit/patch will all the changes, I'm happy to incorporate that. Sorry, I'm a little pressed for time.

Reply:
Most IDEs will allow you to update all at once with F2 (or you can do a ctrl-f-replace). IMO this name as-is is incorrect/misleading since DepositPreauth and Permissioned Domains use different lengths of arrays.

Reply:
Bump

Reply:
[3dfeaf1](https://github.com/XRPLF/xrpl.js/pull/2874/commits/3dfeaf1d68fb42f48f0c86840c4aec4af2af46be)